providers/basic/basic-auth.go

```diff
@@ -1,18 +1,17 @@
-package providers
+package basic
 
 import (
-	"github.com/janekolszak/idp/core"
-	"github.com/janekolszak/idp/helpers"
-
 	"fmt"
-	"golang.org/x/crypto/bcrypt"
 	"net/http"
+
+	"github.com/janekolszak/idp/core"
```

Comment:
Sorry, the tooling I use (goreturns) did it like this.  Not trying to annoy..  I like the way goreturns does this, but it's your project so happy to play by your rules, of course.